### DISCUSSION TITLE
状态栏快捷按钮失效

### DISCUSSION THREAD

**Discussion by fengyuclub** [trojan-gfw/igniter#401] [General]
![IMG_20211225_093917.jpg](https://user-images.githubusercontent.com/18002173/147375588-26927a84-021c-4412-8a0e-cb76daed2956.jpg)

**Comment by fengyuclub** [comment]
卸载清理数据，重装还是一样的问题。

**Comment by TchaikovDriver** [comment]
尝试在应用设置里关掉限制后台、电池优化等功能。

Achophiark ***@***.***>于2021年12月25日 周六上午9:42写道：

**Reply by fengyuclub** [reply]
故障依旧，任务栏灰色，恐怕和节电无关。对了忘记说了，oppo colorOS11.1 安卓V11

**Comment by wongsyrone** [comment]
Good catch. Will investigate the issue.

**Comment by dispcurr** [comment]
Same issue on Huawei HarmonyOS 2.0.0.209(C00E204R2P6patch03), Mate 20 Pro.

**Comment by JiangTian6666** [comment]
miui13也是一样的问题。

**Reply by liutf** [reply]
0.10.4-beta 在miui12.5 上的通知栏按钮也没了

**Comment by Mrshulan** [comment]
> miui13也是一样的问题。

请问，问题有解决吗
